src/main/java/org/gridsuite/explore/server/dto/ElementAttributes.java

```diff
@@ -30,7 +30,7 @@ public class ElementAttributes {
 
     private String owner;
 
-    private long subdirectoriesCount;
+    private Long subdirectoriesCount;
```

Comment:
In **the directory-server**: when updating an element  the  attribute **isAttributesUpdatable** is not  updatable. isAttributesUpdatable should be null otherwise an exception is thrown. 
Check : **/elements/{elementUuid}**  endpoint on the directory server.